cmd/create/admin/cmd.go

```diff
@@ -221,33 +221,20 @@ func FindExistingClusterAdminIDP(cluster *cmv1.Cluster, r *rosa.Runtime) (
 		os.Exit(1)
 	}
 
+	// Locates IDP with name "cluster-admin" that stores the admin user(s)
 	for _, item := range idps {
-		if ocm.IdentityProviderType(item) == ocm.HTPasswdIDPType {
-
+		idpName, idpNameExists := item.GetName()
```

Comment:
do we have a case that idpNameExists is False?

Reply:
I don't believe the name of the IDP will ever be empty as there are validations built in on both the CLI and the backend . Should the check be removed?